module-2/backend/week_two.md

```diff
@@ -26,29 +34,86 @@ end
 
 Now how would you find the owner of the team with an id of 4?
 
-3. What do they allow you to do?
-7. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
-8. Define foreign key, primary key, and schema.
-9. Describe the relationship between a foreign key on one table and a primary key on another table.
-10. What are the parts of an HTTP response?
-11. Describe some techniques to make our Sinatra code more DRY. Give an example of when you would use these techniques.
+* ** Team.find(4).owner**
+
+What do they allow you to do?
+
+* **I don't understand the context of this question.**
+
+In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+
+* **Teachers - 1 ---- n - Students** 
+
+Define foreign key, primary key, and schema.
+
+* **The primary key is the unique identifier for an entry on a table. A foreign key is a primary key held on a different table that establishes the relationship between the two entities. A schema is a scheme/diagram of all the tables and their relationships to each other.**
 
+Describe the relationship between a foreign key on one table and a primary key on another table.
+
+* **See #8**
+
+What are the parts of an HTTP response?
+
+* **The data from the model rendered in the view (or redirected?)**
```

Comment:
HTTP response has two parts: headers, and body. The status code is the first of the headers.